README.md

```diff
@@ -169,6 +169,13 @@ Note that the choice of directory can affect `bkt`'s performance: if the cache
 is stored under a [`tmpfs`](https://en.wikipedia.org/wiki/Tmpfs) or solid-state
 partition it will be significantly faster than caching to a spinning disk.
 
+### Environment Variables
```

Comment:
I think it would be better to document these variables alongside their sibling flags, rather than in a separate section. If you feel this is preferable each variable here should at least link to the relevant section of the readme, and call out which flag it's replacing.

Reply:
> I think it would be better to document these variables alongside their sibling flags

I fear adding this to the main usage section would be overwhelming. Using environment variables instead of command line flags is a niche usage, don't need to bother everyone with this.